app/views/homepage/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<h1>Media Ranker</h1>
+
+<h2>Media Spotlight</h2>
+<% spotlight = @works.find_by(id: 1) %>
+<ul>
+    <li><%= link_to spotlight.title, work_path(spotlight) %></li>
+    <li>Category: <%= spotlight.category %></li>
+    <li>Creator: <%= spotlight.creator %></li>
+    <li>Published in: <%= spotlight.publication_year %></li>
+    <li>Description: <%= spotlight.description %></li>
+</ul>
+
+<h2>Top Albums</h2>
+<ul>
+    <% top_ten = @albums.sample(10) %>
+    <% top_ten.each do |album| %>
+        <li>
+            <%= link_to album.title, work_path(album) %>
+        </li>
+    <% end %>
+</ul>
+
+<h2>Top Books</h2>
```

Comment:
Note that this view code is almost identical to the code above for albums. You can use a loop or a partial view, combined with a model method that selects the work for a particular category, to DRY up this code.